src/Components/Dice.jsx

```diff
@@ -0,0 +1,32 @@
+import './Dice.css'
+import { useState } from "react";
+
+function getRandomFace () {
+    const randomNumber = Math.ceil(Math.random()*6);
+    return randomNumber;
+} //This works
+
+const diceFacesArray = [/* './src/assets/images/dice1.png', './src/assets/images/dice2.png', './src/assets/images/dice3.png', './src/assets/images/dice4.png', './src/assets/images/dice5.png', './src/assets/images/dice6.png' */];
+
+for(let i = 0; i < 6; i++) {
+    diceFacesArray.push(`./src/assets/images/dice${i+1}.png`);
```

Comment:
👀 original approach!

Reply:
jajaj era para no repetir código 😜. Me pasa algo raro cuando envio el pull request, me sale como que solo he hecho la iteration 1 en el feedback automático 🫤 en el resto me pone not done